docs/components.md

```diff
@@ -57,6 +57,39 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used.
+
+To respond to events on a component's model, a parent or ancestor application/component can listen to the event on the model but will have to scope the listening action to the component. Events on the component's model must be exposed to its ancestor via the `exposedEvents` key.
+
+```javascript
+var ClockModel = BaseModel.extend({
+  postInitialize: function() {
+    this.listenTo(this, 'change:time_component:est_time', this.updateTime);
+  },
+
+  updateTime: function() {
+    // update the time
+  }
+});
+```
+
+Shown below is how the `change` event for `est_time` and a custom `tick` events on the `time_component` is exposed.
+
+var TimeModel = BaseModel.extend({
+  exposedEvents: ['change:est_time', 'tick:time_component'],
+  ...
+});
+
+In the above snippet,`time_component` is a child component of the `ClockModel`. We listen for a `change` on the model but scope the listening to `time_component`'s `est_time` property and call `this.updateTime` as our callback method.
+
+If we consider a scenario without a component, we would typically listen for a change on the application model like below
+
+```javascript
+this.listenTo(this.get('time_component'), 'change:est_time', this.updateTime);
+```
+
+The above code works fine until we change `time_component`. For instance, if it gets destroyed and a new one is created then the property `est_time` on which we are listening for a change event becomes no longer valid. This effectively means our callback will never get called.
```

Comment:
I think we need to add some more context to this paragraph...right now it's not clear why we're jumping context from "this is how events work" to "this is why you _should_ let events bubble instead of listening directly on the component".